packages/contracts/contracts/test/ValidatorManager.t.sol

```diff
@@ -736,36 +781,23 @@ contract ValidatorManagerTest is ValidatorSystemUpgrade_Initializer {
         assertTrue(valMgr.getStatus(asserter) == IValidatorManager.ValidatorStatus.REGISTERED);
 
         // Asserter asset decreased by slashingAmount
-        uint128 asserterTotalKro = assetMgr.totalKroAssets(asserter) -
-            kghCounts *
-            kghManager.totalKroInKgh(1);
-        assertEq(asserterTotalKro, minActivateAmount - slashingAmount);
-        assertEq(assetMgr.totalValidatorKro(asserter), asserterTotalKro);
-        // Asserter has 0 rewards
-        assertEq(assetMgr.reflectiveWeight(asserter), assetMgr.totalKroAssets(asserter));
+        uint128 asserterTotalValidatorKro = assetMgr.totalValidatorKro(asserter);
+        assertEq(asserterTotalValidatorKro, minActivateAmount - slashingAmount);
 
         // Security council balance of asset token increased by tax
         uint128 taxAmount = (slashingAmount * assetMgr.TAX_NUMERATOR()) /
             assetMgr.TAX_DENOMINATOR();
         assertEq(assetToken.balanceOf(assetMgr.SECURITY_COUNCIL()), taxAmount);
 
         // Challenger asset increased by output reward and challenge reward
-        // Boosted reward with 100 kgh delegation
-        uint128 boostedReward = 6283173600000736769;
+        // Boosted reward with 1 kgh delegation
+        uint128 boostedReward = mockValMgr.getBoostedReward(challenger);
         uint128 challengeReward = slashingAmount - taxAmount;
-        uint128 challengerAsset = assetMgr.reflectiveWeight(challenger);
+        uint128 challengerKro = assetMgr.totalValidatorKro(challenger);
         assertEq(
-            challengerAsset,
-            minActivateAmount +
-                kghCounts *
-                kghManager.totalKroInKgh(1) +
-                baseReward +
-                boostedReward -
-                1 + // Boosted reward is reduced by 1 when distributed to validator and delegators
-                challengeReward -
-                1 // Challenge reward is reduced by 1 when distributed to each assets in validator vault
+            challengerKro,
+            minActivateAmount + baseReward + (boostedReward / 10) + challengeReward
         );
-        assertGt(assetMgr.totalValidatorKro(challenger), minActivateAmount);
     }
```

Comment:
How about add `revertSlash` test?

Reply:
I thought that it will covered in Colosseum test. BTW, I'll add this too.